lib/mix_test_watch/paths.ex

```diff
@@ -24,6 +24,13 @@ defmodule MixTestWatch.Path do
     |> Enum.any?()
   end
 
+  defp _watching?(path) do
```

Comment:
👍 I've been a bit unsure about this. I've seen it occasionally in other projects, but it's been inconsistent. I'll be sure to stray from this.